tests/ashley/test_templatetags_custom_tags.py

```diff
@@ -160,36 +136,78 @@ def get_html_post_by(user):
         self._init_forum(forum, context)
 
         # Set up topic
-        topic1 = TopicFactory(forum=forum, poster=user1, subject="topic by instructor")
+        topic1 = TopicFactory(
+            forum=forum, poster=user1, subject="Je ne suis pas votre élève"
+        )
 
         # Set up post
         PostFactory.create(
             topic=topic1,
             poster=user1,
-            subject="post by instructor",
+            subject="Vous n'êtes pas mon professeur",
         )
 
+        # log user1
         self.client.force_login(user1)
 
         # accessing forum view
         response = self.client.get(reverse("forum:index"))
+        with self.assertHTML(response, ".forum-last-post") as (html,):
+            extract = parse_html(str(etree.tostring(html)))
```

Comment:
I found a better name than `extract` :
```suggestion
            fragment = parse_html(str(etree.tostring(html)))
```